go/billing/settings.py

```diff
@@ -56,6 +56,11 @@
 LOW_CREDIT_NOTIFICATION_EMAIL = getattr(
     settings, 'BILLING_LOW_CREDIT_NOTIFICATION_EMAIL', 'support@vumi.org')
 
+PROVIDERS = getattr(
+    settings, 'BILLING_PROVIDERS', (
+        ('mtn', 'MTN'),
+        ('vodacom', 'Vodacom')))
```

Comment:
I was thinking of putting these in a separate database table so that admins can add new ones without bugging developers, but happy with that happening in a separate PR.